common/lib/driver-definitions/api-report/driver-definitions.api.md

```diff
@@ -263,7 +262,7 @@ export interface IThrottlingWarning extends IDriverErrorBase {
 // @public (undocumented)
 export interface IUrlResolver {
     // (undocumented)
-    getAbsoluteUrl(resolvedUrl: IResolvedUrl, relativeUrl: string, packageInfoSource?: IFluidCodeDetails | IContainerPackageInfo): Promise<string>;
+    getAbsoluteUrl(resolvedUrl: IResolvedUrl, relativeUrl: string, packageInfoSource?: IContainerPackageInfo): Promise<string>;
```

Comment:
Did you check on partner usages? The change should be fairly simple though. It would be nice if they could leverage on that helper function I mentioned in the other comment.